_members/david-nicholson.md

```diff
@@ -5,15 +5,18 @@ aliases:
 image: images/team/david.jpg
 role: phd
 links:
+  home-page: https://danich1.github.io
   orcid: 0000-0003-0002-5761
-  email: davnic@mail.med.upenn.edu
+  email: dnicholson329@gmail.com
   github: danich1
+  twitter: dnicholson329
 ---
 
-David is a Ph.D. student in the Greene Lab through [Penn’s Genomics and Computational Biology (GCB) PhD Program](https://www.med.upenn.edu/gcb/).
-David graduated from University of Maryland Baltimore County in 2015 with a B.S. in Computer Science and minor in Bioinformatics.
-He joined the lab in summer 2017.
-He was awarded a [T32 Training Grant](https://genomedt32.med.upenn.edu/compgen/people/).
-His research interests include machine learning with a focus on deep learning methods, natural language processing (NLP) and network analysis.
-Currently, his work focuses on using a publicly available resource called [Hetionet](https://het.io/), which is a heterogeneous network that contains pharmacological and biological information.
-He recently submitted a [manuscript](https://greenelab.github.io/text_mined_hetnet_manuscript/) that involved evaluating a text mining framework that to quickly scale up Hetionet.
+David is a rising 6th year Ph.D. candidate at the Unversity of Pennsylvania (Penn) in the <a href="https://www.med.upenn.edu/gcb/">Genomics and Computational Biology</a>program. 
```

Comment:
Why not just use markdown link here? Also missing space between `Biology</a>program`